src/cmd/create.go

```diff
@@ -20,7 +20,9 @@ func init() {
 	rootCmd.AddCommand(createCmd)
 
 	createCmd.PersistentFlags().StringVarP(&createDataFile, "file", "f", "-", "File to read data from. If '.' then reads from './data.yaml'. Defaults to reading from stdin.")
-	viper.BindPFlags(createCmd.Flags())
+	if err := viper.BindPFlags(createCmd.Flags()); err != nil {
```

Comment:
Not sure which is more idomatic go but it could also be like this

```suggestion
	cobra.CheckErr(viper.BindPFlags(createCmd.Flags()))
```

since `cobra.CheckErr` already does that if statement checking for nil.

Reply:
Great point! This is much simpler and achieves the same outcome 👍